designer/server/src/test/scala/pl/touk/nussknacker/ui/api/StatisticsApiHttpServiceBusinessSpec.scala

```diff
@@ -117,12 +121,12 @@ class StatisticsApiHttpServiceBusinessSpec
       given()
         .when()
         .basicAuthReader()
-        .jsonBody("""
+        .jsonBody(s"""
               |{
               | "statistics": [
-              |  {"name": "SEARCH_SCENARIOS_BY_NAME"},
-              |  {"name": "FILTER_SCENARIOS_BY_STATUS"},
-              |  {"name": "SEARCH_SCENARIOS_BY_NAME"}
+              |  {"name": "$SearchScenariosByName"},
```

Comment:
I would leave it as it was or add explicit entryName to show name explicitly (which part of enum is used in API)